### PR TITLE
Make the README.md non-terrible

### PR DESCRIPTION
**Pull request by thebeanogamer**

### CODE REVIEW

**Comment by REDACTED-REDACTED** [comment]
I'm not 100% sure on what syntax is used for the :l13c9 and :lmgtfy commands so they need checking because I think @shadycake changed the syntax.

**Comment by shadycake** [comment]
@thebeanogamer Yup you got that correct, and the `:lxcx` commands are now just `:level x x` or optionally `:l` and `:lc` are aliases to `:level`

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
...
+* **`:help`** - Displays information about the usage and syntax of the commands
+
+### Cyber Security Commands:
+* **`:TODO`** - Gets information about a specific CyberStart Game challenge.
+* **`:haveibeenpwned email@exmaple.com`** - Searches haveibeenpwned.com for breached accounts.
+* **`:hasitbeenpwned password`** - Searches pwnedpasswords.com for breached passwords.
+
+### Commands for Fun:
+* **`:lmgtfy Stupid Question?`** - Returns a LMGTFY URL for the given question.
+* **`:react OMG`** - Reacts to the previous message with an emoji version of the given string.
```
Comment (shadycake) [inline README.md]: The react command currently doesn't work like this, it just takes space separated emojis. This is on the to-do list to be reimplemented.

Comment (thebeanogamer) [reply README.md]: Please write that in

File: README.md [hunk 2]
```diff
@@ -1,2 +1,16 @@
...
+* **`:help`** - Displays information about the usage and syntax of the commands
+
+### Cyber Security Commands:
+* **`:level 13 1`** - Gets information about a specific CyberStart Game challenge.
+* **`:haveibeenpwned email@exmaple.com`** - Searches haveibeenpwned.com for breached accounts.
+* **`:hasitbeenpwned password`** - Searches pwnedpasswords.com for breached passwords.
+
+### Commands for Fun:
+* **`:lmgtfy [-d][-ie] Stupid Question?`** - Returns a LMGTFY URL for the given question.  Adding `-d` will delete the message that instigated the command and `-ie` will enable the internet explainer feature on lmgtfy.
+* **`:react :question:`** - Reacts to the previous message with the space seperated emojis in the requesting message.
```
Comment (shadycake) [inline README.md]: `:react :question:`?

Comment (REDACTED-REDACTED) [reply README.md]: That's how you do discord emojis

Comment (shadycake) [reply README.md]: This isn't discord, and even still it won't show up as :question: because you have it in an in-line code block `:question:`

**Pull request merged**